scripts/docker_build_release.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+RELEASE_TAG ?= $(CIRCLE_TAG)
```

Comment:
This looks like Makefile syntax. It should be this:

```
RELEASE_TAG=$CIRCLE_TAG
```

Also anywhere else where variables are being used, the syntax `$(VARIABLE_NAME)` needs to be `$VARIABLE_NAME`. With the first one, bash tries to evaulate whatever is inside as a command :)